exercises/L-functions-nested/exercise.js

```diff
@@ -1,5 +1,16 @@
+
```

Comment:
That is really nice job Meron. your code is simple, short and understandable. Just as I read in the instruction of this part of exercise, we should write **two functions**. one for change the names to uppercase and another for greeting. I think if you just add this function to your code it will be perfect.

Reply:
okay, I will take your comment and review the code again and fix it. Thank you very much.